packages/@o3r/rules-engine/schematics/index.it.spec.ts

```diff
@@ -31,7 +31,8 @@ describe('ng add rules-engine', () => {
     expect(diff.added).toContain('apps/test-app/cms.json');
     expect(diff.added).toContain('apps/test-app/placeholders.metadata.json');
     expect(diff.added).toContain('apps/test-app/tsconfig.cms.json');
-    expect(diff.added.length).toBe(12);
+    expect(diff.added).toContain('apps/test-app/migration-scripts/README.md');
+    expect(diff.added.length).toBe(13);
```

Comment:
What this number comes from? We have the 4 expected files, but what about the others? Could it be something we could compute ?
(eg: expectedFileList = ['app/test-app/cms.json', etc.]
expectedFileList.forEach((file) => expect(diff.added).toContain(file)))
expect(diff.added.length).toBe(? + expectedFileList.length) 
?